### ISSUE TITLE
when accessibility active, list items containing only links require two presses of the tab key

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2040]
### Subject of the issue/enhancement/features
If you create a list item that consists of just a hyperlink, the user must press tab twice to navigate it when accessibility is active

### Your environment
* framework 3.0.0
* any browser

### Steps to reproduce
Add the following to the `body` of a component:
```html
These <a href='http://www.adaptlearning.org' target='_blank'>presentation components</a> can be structured in any way to help you create learning experiences that meet the needs of your learners.<ul><li><a href='https://www.kineo.com/' target='_blank'>kineo</a></li><li>this is not a link</li><li><a href='https://www.google.com/' target='_blank'>google</a></li></ul>
```

### Expected behaviour
I should be able to navigate past the links in the unordered list with a single tab keypress

### Actual behaviour
I have to press the tab key twice for links in the unordered list

### Screenshots (if you can)
![double-tabbing](https://user-images.githubusercontent.com/1249597/38825720-89fface4-41a5-11e8-8a7f-7cf1250fa5f5.png)

**Comment by oliverfoster** [comment]
Remove a11y_text helper, merged
